docs/versions/latest/Release.md

```diff
@@ -30,10 +30,21 @@ SQLDatabaseManager.getDatabase() function is removed in vRA 7.6 / Aria Automatio
 [//]: # (Improvements -> Bugfixes/hotfixes or general improvements)
 ## Improvements
 
+### Blueprints with the same name in a single project will give out a better error message now
+
+#### Previous Behaviour
+
+The error was misleading, something like DuplicateKey@VraNgBluerpintSHA was given, since we tried to assign a value to a map
```

Comment:
In general I would recommend to use present tense in documentation, something like "Misleading error is thrown when duplicate blueprint names are found..." or something like that
But the current approach is also ok

Reply:
Done, good point